dashboard/templates/assignments.html

```diff
@@ -23,9 +23,14 @@ <h5 style="padding-left: 10px">Progress toward Final Grade</h5>
         <div class="column">
             <h5>Assignments Due By Date</h5>
             <span>Show assignments that weigh at least</span>
+            {% verbatim %}
             <p ng-show="percentFilterEnabled">
                 <select ng-change="updatePercentFilter()" ng-model="percent_filter" ng-options="obj.value as obj.text for obj in percent"></select>
+                &nbsp;
+                <input ng-show="isResetMyDefaultCheckboxEnabled" type="checkbox" ng-model="default_selection_model" ng-change="sendDefaults()">
+                <label>{{default_setting.msg}}</label>
```

Comment:
Does this label print correct within the verbatim tags? You might need to move the endverbatim up a little. Not 100% sure why it's needed here? Maybe you can add some comments?

Reply:
Django itself uses tags like this `{% endblock %}` which are placed in the curly braces. Django for some reason get confused if don't have angular tags `{{}}`in verbatim tags.

Reply:
Interesting, maybe I never saw it used. 

Perhaps we should look at the [django-angular](https://github.com/jrief/django-angular) package which has an [{% angularjs %} tag.
](https://django-angular.readthedocs.io/en/latest/template-sharing.html)

Reply:
I did not look into this so deeply at this during winter pilot. I am reluctant to use any more libraries supporting angularJS as it is old, entered into [long term support](https://blog.angular.io/stable-angularjs-and-long-term-support-7e077635ee9c) and we should move away from it for `assignment` view and some time in future this version won't be supported and it will become ancient.